recipes/orange3-timeseries/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: orange3-timeseries
+  version: "0.2.5"
+
+source:
+  fn: Orange3-Timeseries-0.2.5.tar.gz
+  url: https://pypi.python.org/packages/ed/7d/d6b6a2b3ec051eb5a10f5e99b9459a51572f1c4bf3aacd72ba2e9a5a0d77/Orange3-Timeseries-0.2.5.tar.gz
+  md5: 09f3d1c95674683aa53d98df9b539bf5
```

Comment:
Shiiit. Using `conda skeleton`, I must have misassumed state of the art output. Will do. Thanks!